app/components/contact/contact-item/contact-item-modal.hbs

```diff
@@ -0,0 +1,101 @@
+<Auk::Modal @size="small">
+  <Auk::Modal::Header
+    @title="Contactpersoon toevoegen"
+    @onClose={{@onCancel}}
+  />
+  <Auk::Modal::Body>
+    <div class="auk-form-group-layout auk-form-group-layout--standard">
+      <div class="auk-form-group">
+        <Auk::Label for="givenName">
+          Voornaam
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="givenName"
+          @block={{true}}
+          value={{this.contact.givenName}}
+          {{on "input" (fn this.setInputValue "givenName")}}
+          {{on "change" this.setFullName}}
+        />
+      </div>
+      <div class="auk-form-group">
+        <Auk::Label for="familyName">
+          Achternaam
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="familyName"
+          @block={{true}}
+          value={{this.contact.familyName}}
+          {{on "input" (fn this.setInputValue "familyName")}}
+          {{on "change" this.setFullName}}
+        />
+      </div>
+      <div class="auk-form-group">
+        <Auk::Label for="mailAddress">
+          E-mailadres
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="mailAddress"
+          @block={{true}}
+          value={{this.contact.mailAddress.value}}
+          {{on "input" (fn this.setRelationInputValue "mailAddress")}}
+        />
+      </div>
+      <div class="auk-form-group">
+        <Auk::Label for="telephone">
+          Telefoon
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="telephone"
+          @block={{true}}
+          value={{this.contact.telephone.value}}
+          {{on "input" (fn this.setRelationInputValue "telephone")}}
+        />
+      </div>
+      <div class="auk-form-group">
+        <Auk::Label for="organizationName">
+          Organisatie
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="organizationName"
+          @block={{true}}
+          value={{this.contact.organizationName}}
+          {{on "input" (fn this.setInputValue "organizationName")}}
+        />
+      </div>
+    </div>
+  </Auk::Modal::Body>
+  <Auk::Modal::Footer @onCancel={{@onCancel}}>
+    <div class="auk-o-flex auk-o-flex--horizontal-align">
+      {{#if @contact}}
+        <Auk::Button
+          class="auk-u-mr"
+          @skin="danger-primary"
+          {{on "click" (fn @onDeleteContact this.contact)}}
```

Comment:
Je geeft hier `this.contact` mee als argument, maar gebruikt dat argument niet in de action.